components/config-mgmt-service/integration_test/node_export_test.go

```diff
@@ -758,15 +759,86 @@ func TestNodeExportPaging(t *testing.T) {
 		data = append(data, tdata.GetContent()...)
 	}
 
-	actualNumberOfNodes := 0
-	dec := json.NewDecoder(bytes.NewReader(data))
-	for dec.More() {
-		var nodes []interface{}
-		err := dec.Decode(&nodes)
-		require.NoError(t, err)
+	var outputNodes []interface{}
+	err = json.Unmarshal(data, &outputNodes)
+
+	require.NoError(t, err)
+
+	assert.Equal(t, numberOfNodes, len(outputNodes))
+}
```

Comment:
This change switches from the lenient json stream parser to the more strict json document parser. The original code before this PR passed with the streaming parser but not with the document parser.